presto-hive/src/main/java/com/facebook/presto/hive/HiveConnectorFactory.java

```diff
@@ -111,6 +112,7 @@ public Connector create(String catalogName, Map<String, String> config, Connecto
                     new HiveSecurityModule(),
                     new HiveAuthenticationModule(),
                     new HiveProcedureModule(),
+                    new CachingModule(),
```

Comment:
IIRC, having a `CachingModule`  injected completely independently could have an issue. I could be wrong. I can't remember exactly but just to raise a potential issue.

Reply:
Shall we move the binding to `HiveClientModule` instead?